src/containerapp/azext_containerapp/_help.py

```diff
@@ -1258,7 +1258,7 @@
           az containerapp env java-component spring-cloud-config create -g MyResourceGroup \\
               -n MyJavaComponentName \\
               --environment MyEnvironment \\
-              --configuration PropertyName1=Value1 PropertyName2=Value2
+              --set-configurations PropertyName1=Value1 PropertyName2=Value2
```

Comment:
If we use `--set-configurations` to update the config, when should we use `--replace-configurations`? What's the difference?